### DISCUSSION TITLE
Helping with Issue #2116

### DISCUSSION THREAD

**Discussion by TomerPacific** [cake-build/cake#3498] [General]
Hi,
following what is written under the Contributing section, I found [this issue](https://github.com/cake-build/website/issues/2116) and I'd like to help out.

Anything I should know?

**Comment by pascalberger** [comment]
Thank you for stepping up 👍 I added a comment with additional information to https://github.com/cake-build/website/issues/2116 which hopefully should get you started. Let me know if you have any further questions and need more information.

@TomerPacific Can you please also add a quick comment to the issue, so that I can assign the issue to you.
